src/state-summary/state-card-content.html

```diff
@@ -35,18 +35,32 @@
     'inputChanged(hass, inDialog, stateObj)',
   ],
 
-  inputChanged: function (hass, inDialog, stateObj) {
-    if (!stateObj) return;
-
+  _inputChanged: function (hass, inDialog, stateObj, stateType) {
     window.hassUtil.dynamicContentUpdater(
       this,
-      ('STATE-CARD-' +
-       window.hassUtil.stateCardType(this.hass, stateObj).toUpperCase()),
+      ('STATE-CARD-' + stateType.toUpperCase()),
       {
         hass: hass,
         stateObj: stateObj,
         inDialog: inDialog,
       });
   },
+
+  inputChanged: function (hass, inDialog, stateObj) {
+    var stateType;
+    if (!stateObj) return;
+    stateType = window.hassUtil.stateCardType(this.hass, stateObj);
+    if (stateType.indexOf('custom') !== -1) {
+      this.importHref(
+          '/local/' + stateType + '.html',
```

Comment:
I think that it would make it more future proof if you have them scope it to a dir. `/local/custom-card/X.html` maybe?